fix-uppercase-lowercase.py

```diff
@@ -62,6 +62,10 @@ def main():
     did_something1 = fix_image_files()
     did_something2 = fix_image_links()
     if did_something1 or did_something2:
+        print(
+            "ERROR: All image filenames must be lowercase, or the build will fail!" +
+            "Please locally make all your image filenames lowercase, and then commit and push again."
```

Comment:
Deze check wordt ook lokaal gedraaid door `docker-compose up`. In dat geval is de verwijzing "commit en push weer" verwarrend.
Op zich is het al enorm verbeterd als je alleen de eerste zin print.

Reply:
Goed punt, ik heb het iets aangepast.